engine/access/apiproxy/access_api_proxy.go

```diff
@@ -201,6 +199,18 @@ func (h *FlowAccessAPIRouter) GetTransactionResult(context context.Context, req
 	return res, err
 }
 
+func (h *FlowAccessAPIRouter) GetTransactionResultsByBlockID(context context.Context, req *access.GetTransactionsByBlockIDRequest) (*access.TransactionResultsResponse, error) {
```

Comment:
These were missing without anyone noticing because of type composition the router still implemented the method.